mmv1/third_party/terraform/resources/resource_iam_binding.go.erb

```diff
@@ -130,8 +131,8 @@ func resourceIamBindingRead(newUpdaterFunc newResourceIamUpdaterFunc) schema.Rea
 		if err != nil {
 			return handleNotFoundError(err, d, fmt.Sprintf("Resource %q with IAM Binding (Role %q)", updater.DescribeResource(), eBinding.Role))
 		}
-		log.Printf("[DEBUG] Retrieved policy for %s: %+v", updater.DescribeResource(), p)
-		log.Printf("[DEBUG] Looking for binding with role %q and condition %+v", eBinding.Role, eCondition)
+		log.Print(spew.Sprintf("[DEBUG] Retrieved policy for %s: %#v", updater.DescribeResource(), p))
```

Comment:
Was this just for debugging implementation? Will it be useful going forward?

Reply:
referring to the use of `spew` specifically

Reply:
No worries just making sure it wasn't leftover from testing

Reply:
Thanks! yeah, it totally is, but for once it's intentional 😂